docs/user-guide/supported-protocols/postgresql.md

```diff
@@ -1,5 +1,14 @@
 # PostgreSQL
 
+PostgreSQL wire protocol is supported by GreptimeDB, which means you can use
+standard PostgreSQL client to connect to DB instances and execute
+queries. Note that when writing SQL queries, GreptimeDB follows MySQL dialect
+instead of PostgreSQL.
+
+Both simple and extended query subprotocols are supported, except for those
+clients relies on parameter type inference, like rust-postgres, will be
+supported later.
+
 ## Using psql
 
 Currently we have PostgreSQL simple query subprotocol supported on
```

Comment:
```suggestion
As for PostgreSQL, only the simple query subprotocol is supported by 
```

Reply:
I just removed this line because we don't have to emphases the simple query support any more, as we already supported both